public/script.js

```diff
@@ -0,0 +1,123 @@
+//tmp_list = ['apple', 'orange', 'grape', 'strawberry', 'apples'];//テスト用
+let created_sentence = [];
+
+const container = document.getElementById('container');
+
+window.addEventListener('load', (event) => {
+    // ウィンドウが読み込まれた時の処理
+    fetch('/api/cocktail/all')
+        .then(response => {
+            if (!response.ok) {
+                throw new Error('Network response was not ok');
+            }
+            //return response.json(); // JSON形式のレスポンスをパース
+            console.log(response);
+        })
+        .then(data => {
+            console.log('Received data:', data); // 取得したデータをコンソールに出力
+            // ここで、データを使った他の処理を実行することができます
+        })
+        .catch(error => {
+            console.error('Error fetching data:', error); // エラーハンドリング
+        });
+});
+
+
+function buttonclick(obj){
+    let text = '';
+    obj.childNodes.forEach(node => {
+        if(node.nodeType === node.TEXT_NODE){
+            text += node.textContent.trim();
+        }
+    });
+    const b_text = text;//クリックしたボタンの文字
+    //console.log(b_text);
+
+
+    if(created_sentence.includes(b_text)){
+        //nothing
+    }else if(document.getElementById("c_button").childElementCount >= 4){//配列の要素で判定
+        alert("これ以上言葉を追加できません");
+    }else{
+        created_sentence.push(b_text);
+
+        //console.log(created_sentence);//fortest
+
+        const newLi = document.createElement("li");
+        newLi.textContent = b_text;
+
+        const button = document.createElement('button');
+        const span = document.createElement('span');
+        span.className = 'material-symbols-outlined';
+        span.textContent = 'close';
+        button.appendChild(span);
+        newLi.appendChild(button);
+
+        const ul = document.querySelector('#clicked_button ul');
+        ul.appendChild(newLi);
+    }
+}
+
+function updatecontainersize(){
+    var buttonstyle = window.getComputedStyle(document.getElementById('button'), '');
```

Comment:
```suggestion
    const buttonstyle = window.getComputedStyle(document.getElementById('button'), '');
```

javascriptでvarは非推奨になってます。
変数はletを使って欲しいし、今回は宣言後に書き換えないと思うので constで良いかなと思いました。